src/types.web.ts

```diff
@@ -0,0 +1,118 @@
+// === Needed imports
+import { basename } from "https://deno.land/std@0.170.0/path/mod.ts";
+import { type InputFileProxy } from "https://esm.sh/@grammyjs/types@2.10.2";
+import { toRaw } from "./platform.deno.ts";
+
+// === Export all API types
+export * from "https://esm.sh/@grammyjs/types@2.10.2";
+
+/** Something that looks like a URL. */
+interface URLLike {
+    /**
+     * Identifier of the resource. Must be in a format that can be parsed by the
+     * URL constructor.
+     */
+    url: string;
+}
+
+// === InputFile handling and File augmenting
+/**
+ * An `InputFile` wraps a number of different sources for [sending
+ * files](https://grammy.dev/guide/files.html#uploading-your-own-file).
+ *
+ * It corresponds to the `InputFile` type in the [Telegram Bot API
+ * Reference](https://core.telegram.org/bots/api#inputfile).
+ */
+export class InputFile {
+    private consumed = false;
+    private readonly fileData: ConstructorParameters<typeof InputFile>[0];
+    /**
+     * Optional name of the constructed `InputFile` instance.
+     *
+     * Check out the
+     * [documentation](https://grammy.dev/guide/files.html#uploading-your-own-file)
+     * on sending files with `InputFile`.
+     */
+    public readonly filename?: string;
+    /**
+     * Constructs an `InputFile` that can be used in the API to send files.
+     *
+     * @param file A path to a local file or a `Buffer` or a `ReadableStream` that specifies the file data
```

Comment:
Docs are wrong here

Reply:
I see. What about this?

```suggestion
     * @param file A URL of a file or a `Blob` or any form of `Uint8Array` that specifies the file data
```